### PR TITLE
9 - Text area which minimizes on paste

### PR DESCRIPTION
**Pull request by CJGutz**
closes: #9

## Proposed changes
Creates a component which minimizes when input is pasted in. 
When clicked again it expands. Expands to the previously defined height.
Created story. Exposed functions to use an external button which can request to minimize the input.

## Screenshots
Sorry not sorry for crusty video :D
vc-minimize

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added
- [ ] Documentation have been written

### CODE REVIEW

File: src/app/components/vc-inspection/MinimizationTextArea.cy.tsx
Comment (thomsen85) [inline src/app/components/vc-inspection/MinimizationTextArea.cy.tsx:1]: What do you think about having this in a __tests__ folder instead?
```
mv MinimizationTextArea.cy.tsx __tests__/
```

File: src/app/components/vc-inspection/MinimizingTextArea.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      onMouseDown={() => setMinimizedAndTransition(false, false)}
+      onMouseUp={() => setMinimizedAndTransition(false, true)}
+      onFocus={() => setMinimizedAndTransition(false, true)}
+      onPaste={() => setMinimizedAndTransition(true, true)}
+      style={{ height: minimized ? MINIMIZED_HEIGHT + 'px' : heightNotMinimized + 'px' }}
+      placeholder="Paste your verifiable credential here"
+      className={cn(
+        'rounded-md bg-light-gray p-4 placeholder-readable-gray',
+        activeTransition && 'transition-all duration-200',
+        minimized && ' text-readable-gray',
```
Comment (thomsen85) [inline src/app/components/vc-inspection/MinimizingTextArea.tsx:82]:
```suggestion
        minimized && ' overflow-hidden text-readable-gray',
```

Comment (thomsen85) [reply src/app/components/vc-inspection/MinimizingTextArea.tsx:82]: Removes the scrollbar on minimize, found this to be a little better

**Review by thomsen85 — APPROVED** [review]
Small Changes, If fixed merge

File: src/app/components/vc-inspection/MinimizationTextArea.cy.tsx
Comment (CJGutz) [reply src/app/components/vc-inspection/MinimizationTextArea.cy.tsx:1]: Agreed

File: src/app/components/vc-inspection/MinimizingTextArea.tsx (hunk 1 above)
Comment (CJGutz) [reply src/app/components/vc-inspection/MinimizingTextArea.tsx:82]: Yea nice. It is not like it can really be used anyways.

**Pull request merged**